apps/api/src/routes/community/community.ts

```diff
@@ -0,0 +1,141 @@
+    try {
+      const { orgId } = c.req.valid('query');
+
+      const result = await fetchCommunityQuestions(orgId);
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to load community questions');
+    }
+  })
+  // --- get single community question
+  .get('/question', authMiddleware, zValidator('query', ZCommunityQuestion), async (c) => {
+    try {
+      const { slug } = c.req.valid('query');
+
+      const result = await fetchCommunityQuestion(slug);
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to load community question');
+    }
+  })
+  // --- add community question
+  .post('/question', authMiddleware, zValidator('json', ZNewCommunityQuestion), async (c) => {
+    try {
+      const { title, body, course_id, organization_id, author_profile_id, votes, slug } = c.req.valid('json');
+
+      const result = await handleAddQuestion({
+        title,
+        body,
+        course_id,
+        organization_id,
+        author_profile_id,
+        votes,
+        slug
+      });
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to add community question');
+    }
+  })
+  // --- update community question
+  .put('/question', authMiddleware, zValidator('json', ZCommunityQuestionUpdate), async (c) => {
+    try {
+      const { id, title, body, course_id } = c.req.valid('json');
+
+      const result = await handleUpdateQuestion({ id: Number(id), title, body, course_id });
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to update community question');
+    }
+  })
+  // --- delete community question (by id)
+  .delete('/question', authMiddleware, zValidator('json', ZCommunityQuestionDelete), async (c) => {
+    try {
+      const { id } = c.req.valid('json');
+
+      const result = await handleDeleteQuestion({ id: Number(id) });
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to delete community question');
+    }
+  })
+
+  // --- add comment
+  .post('/comment', authMiddleware, zValidator('json', ZCommunityComment), async (c) => {
+    try {
+      const { body, questionId, authorId, votes } = c.req.valid('json');
+
+      const result = await handleSubmitComment({
+        body,
+        questionId: Number(questionId),
+        authorId: String(authorId),
+        votes: Number(votes)
+      });
+
+      return c.json({ success: true, data: result }, 201);
+    } catch (error) {
+      return handleError(c, error, 'Failed to submit comment');
+    }
+  })
+  // --- upvote comment/question
+  .put('/comment', authMiddleware, zValidator('json', ZUpvoteComment), async (c) => {
```

Comment:
Please go over my message here with the route naming and go over all the other routes. So I don't have to comment on each one.

https://discord.com/channels/1169524228635238463/1447514657165283440/1447720816954445965